### PR TITLE
[datafeeder]: INSPIRE compliance

### PR DESCRIPTION
**Pull request by fgravin**
Update metadata template to be INSPIRE compliant
See https://github.com/inspire-eu-validation/metadata/blob/2.0/common/conditions-for-access-and-use.md

### CODE REVIEW

**Comment by fgravin** [comment]
@pmauduit could you plz review and merge if it's ok.
I'm not sure about the workflow, i did this PR in `master`, how should i reverse this in `docker-master` ?

**Comment by pmauduit** [comment]
the workflow is correct (commits should go first into master, then are merged back into docker-master).

File: datafeeder/metadata_template.xml [hunk 1]
```diff
@@ -143,17 +143,13 @@
...
-              codeListValue="otherRestrictions" />
-          </gmd:accessConstraints>
           <gmd:useConstraints>
-            <gmd:MD_RestrictionCode
-              codeList="http://standards.iso.org/iso/19139/resources/gmxCodelists.xml#MD_RestrictionCode" codeListValue="license" />
+            <gmd:MD_RestrictionCode codeList="http://standards.iso.org/iso/19139/resources/gmxCodelists.xml#MD_RestrictionCode" codeListValue="otherRestrictions"/>
           </gmd:useConstraints>
           <gmd:otherConstraints>
-            <gmx:Anchor xlink:href="http://inspire.ec.europa.eu/metadata-codelist/LimitationsOnPublicAccess/noLimitations">No limitations to public access</gmx:Anchor>
+            <gmx:Anchor xlink:href="http://inspire.ec.europa.eu/metadata-codelist/ ConditionsApplyingToAccessAndUse/noConditionsApply">
```
Comment (pmauduit) [inline datafeeder/metadata_template.xml:150]: space in the url ?

Comment (fgravin) [reply datafeeder/metadata_template.xml:150]: yes weird it comes from the official INSPIRE guide, and it seems to pass the validation but I will fix that.

**Pull request merged**